osu.Game.Rulesets.Osu/Utils/OsuHitObjectGenerationUtils.cs

```diff
@@ -100,5 +104,53 @@ public static Vector2 RotateVectorTowardsVector(Vector2 initial, Vector2 destina
                 initial.Length * MathF.Sin(finalAngleRad)
             );
         }
+
+        /// <summary>
+        /// Reflects an OsuHitObject's position horizontally (over the 'y axis').
```

Comment:
I think this mention of the "y axis" is pretty confusing. I almost went to reflexively correct it to "x axis". "Horizontally/vertically" is enough I think.